### PR TITLE
Link to external Azure IoT Hub component - iothub-react

### PR DESCRIPTION
**Pull request by dluc**
I'd like to add "iothub-react" to Alpakka, it's a component to stream messages from/to IoT devices using Microsoft Azure IoT Hub. 

Source code and documentation are available here https://github.com/Azure/toketi-iothubreact and the project is also linked here https://azure.github.io/projects/sdks . 

I tested the page rendering locally (thanks @johanandren for the instructions)

Please let me know if there's anything we can do to improve :-)

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @dluc,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by johanandren** [comment]
LGTM, if signing the CLA is a problem I can PR the same link.

**Comment by dluc** [comment]
Shouldn't be a problem, I'd like to get the CLA done so we're good to go
also with future contributions.
A matter of a few days until I get approval internally.

Thanks!

File: docs/src/main/paradox/external-connectors.md [hunk 1]
```diff
@@ -25,4 +25,8 @@ TODO: Add links to more connectors. [Contributions](https://github.com/akka/alpa
 
 [Streaming TCP Java documentation](http://doc.akka.io/docs/akka/2.4/java/stream/stream-io.html#Streaming_TCP)
 
-[Streaming TCP Scala documentation](http://doc.akka.io/docs/akka/2.4/scala/stream/stream-io.html#Streaming_TCP)
\ No newline at end of file
+[Streaming TCP Scala documentation](http://doc.akka.io/docs/akka/2.4/scala/stream/stream-io.html#Streaming_TCP)
+
+## Azure
```
Comment (patriknw) [inline docs/src/main/paradox/external-connectors.md]: is it a connector to Azure or shall we be more specific? Azure IoT Hub?

Comment (dluc) [inline docs/src/main/paradox/external-connectors.md]: specifically it is a connector to Azure IoT Hub underlying storage. Do you think the section title should be more specific ? My reasoning about using a generic "Azure" category is to group future connectors (e.g. Azure tables, Azure service bus, etc) under the same umbrella. At the same time we could also differentiate though, and have Azure Storage and Azure IoT. I'm open to suggestions.

Comment (patriknw) [inline docs/src/main/paradox/external-connectors.md]: Thanks for clarifying. In that light I think `Azure` is fine.

**Comment by dluc** [comment]
CLA sorted out, good to go

**Pull request merged**